cmake/DARTConfig.cmake.in

```diff
@@ -19,6 +19,23 @@ set(DART_LIBRARY_DIRS "@CMAKE_INSTALL_PREFIX@/lib" "@Boost_LIBRARY_DIRS@")
 
 include("${CURRENT_DIR}/DARTTargets.cmake")
 
-set(DART_LIBRARIES dart)
+find_library(dart_location NAMES "dart" PATHS "@CMAKE_INSTALL_PREFIX@/lib")

```

Comment:
This wouldn't work if DART is built in debug mode since it contains appendix "d". I'm not sure we should care the case.

Also, I haven't seen quotation mark is used for library name in find_library. Do we need it?